test/puppetlabs/kitchensink/core_test.clj

```diff
@@ -731,3 +731,9 @@
         (is (= false
                (with-timeout 1 false
                  (wait-return 1005 true))))))))
+
+(deftest open-port-num-test
+  (let [port-in-use (open-port-num)]
+    (with-open [s (java.net.ServerSocket. port-in-use)]
+      (let [open-ports (set (take 60000 (repeatedly open-port-num)))]
+        (is (not (contains? open-ports port-in-use)))))))
```

Comment:
nice - i was just thinking of a test that asserts it returns an integer or something, but this is way more thorough :)

think it would make sense to add `(is (every? pos? open-ports))`?  then I'm good.